superset-frontend/src/components/Label/index.tsx

```diff
@@ -92,6 +92,7 @@ export default function Label(props: LabelProps) {
 
   return (
     <Tag
+      role="button"
```

Comment:
We could apply the same logic of checking for an onClick prop on the Tag component too